styles/base.css

```diff
@@ -0,0 +1,11 @@
+body {
+    width: 480px;  
+    background: black;
+    margin: 0px auto;
+}
+
+div {
+    float: left;

```

Comment:
float directly affects the 'layout' of the <div>'s so this style could have gone in layout.css